packages/nextjs/app/page.tsx

```diff
@@ -19,6 +20,77 @@ const Home: NextPage = () => {
     }
   }
 
+  const { address } = useAccount();
+
+  const { data: Ens } = useEnsName({
+    address,
+    chainId: 1,
+  });
+
+  const { data: allowStatus } = useScaffoldContractRead({
+    contractName: "BatchRegistry",
+    functionName: "allowList",
+    args: [address],
+  });
+
+  const { data: checkedInStatus } = useScaffoldContractRead({
+    contractName: "BatchRegistry",
+    functionName: "yourContractAddress",
+    args: [address],
+  });
+  const CheckStatus = useCallback(() => {
```

Comment:
Let's create a separate component for this, we can move all the necessary hooks in that component itself